test/Microsoft.AspNetCore.Mvc.DataAnnotations.Test/Internal/CompareAttributeAdapterTest.cs

```diff
@@ -27,8 +27,7 @@ public void ClientRulesWithCompareAttribute_ErrorMessageUsesDisplayName_WithoutL
             var adapter = new CompareAttributeAdapter(attribute, stringLocalizer: null);
 
             // Mono issue - https://github.com/aspnet/External/issues/19
-            var expectedMessage = PlatformNormalizer.NormalizeContent(
-                    "'MyPropertyDisplayName' and 'OtherPropertyDisplayName' do not match.");
+            var expectedMessage = "'MyPropertyDisplayName' and 'OtherPropertyDisplayName' do not match.";
```

Comment:
BTW this part of the change  seems unrelated and unnecessary. You can make TestCommon a regular project without removing `PlatformNormalizer.NormalizeContent()`.